src/itemlistformaction.cpp

```diff
@@ -969,39 +969,49 @@ void ItemListFormAction::do_update_visible_items()
 
 void ItemListFormAction::draw_items()
 {
-	const unsigned int width = list.get_width();
 	auto datetime_format = cfg->get_configvalue("datetime-format");
 	auto itemlist_format =
 		cfg->get_configvalue("articlelist-format");
 
-	switch (invalidation_mode) {
-	case InvalidationMode::COMPLETE:
-		listfmt.clear();
-
-		for (const auto& item : visible_items) {
-			auto line = item2formatted_line(item,
-					width,
-					itemlist_format,
-					datetime_format);
-			listfmt.add_line(line);
-		}
-		break;
-
-	case InvalidationMode::PARTIAL:
-		for (const auto& itempos : invalidated_itempos) {
-			auto item = visible_items[itempos];
-			auto line = item2formatted_line(item,
-					width,
-					itemlist_format,
-					datetime_format);
-			listfmt.set_line(itempos, line);
+	//switch (invalidation_mode) {
+	//case InvalidationMode::COMPLETE:
+	//	listfmt.clear();
+
+	//	for (const auto& item : visible_items) {
+	//		auto line = item2formatted_line(item,
+	//				width,
+	//				itemlist_format,
+	//				datetime_format);
+	//		listfmt.add_line(line);
+	//	}
+	//	break;
+
+	//case InvalidationMode::PARTIAL:
+	//	for (const auto& itempos : invalidated_itempos) {
+	//		auto item = visible_items[itempos];
+	//		auto line = item2formatted_line(item,
+	//				width,
+	//				itemlist_format,
+	//				datetime_format);
+	//		listfmt.set_line(itempos, line);
+	//	}
+	//	break;
+	//case InvalidationMode::NONE:
+	//	break;
+	//}
```

Comment:
I agree, it would be nice to get rid of the `InvalidationMode`.
At this moment, it is still used to check when the list's line-cache should be cleared.